ui/apps/everest/src/consts.ts

```diff
@@ -16,9 +16,10 @@ import { BackupStatus } from 'shared-types/backups.types';
 
 // limitations under the License.
 export const IP_REGEX =
-  /^([0-9]{1,3}\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2]))?$/;
+  /^([0-9]{1,3}\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2])){1}$/;
 
-export const INVALID_SOURCE_RANGE_ERROR = 'Invalid IP address range';
+export const INVALID_SOURCE_RANGE_ERROR =
+  'Provide an IP address along with its corresponding netmask. For example, 255.255.255.0/32.';
 export const SOURCE_RANGE_PLACEHOLDER =
   'Please enter the single IP address or range using netmask (e.g. 192.168.1.1/24)';
```

Comment:
Leaving a comment just to ensure we don't forget to change this message.

Reply:
I modified both the placeholder and the error message to say 
`Please enter a single IP address (e.g., 192.168.1.1/32) or a network range (e.g., 192.168.1.0/24).`